db/migrate/20200412021713_add_initial_logistics_tables.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class AddInitialLogisticsTables < ActiveRecord::Migration[6.0]
+  def change
+    create_table :printer_models do |t|
+      t.string :name
+      t.integer :x_mm
+      t.integer :y_mm
+      t.integer :z_mm
+      t.boolean :petg
+      t.boolean :abs
+      t.timestamps
+    end
+
+    create_table :addresses do |t|
+      t.string :line_1
+      t.string :line_2
+      t.string :locality
+      t.string :administrative_area
+      t.string :postal_code
+      t.string :country
+      t.string :lat
+      t.string :lon
```

Comment:
Should we consider enabling PostGIS for this and and a `t.st_point :lonlat`? https://github.com/rgeo/activerecord-postgis-adapter